packages/lab/stories/pill-next/pill-next.stories.tsx

```diff
@@ -22,6 +25,40 @@ export const Disabled: StoryFn<typeof PillNext> = () => {
   );
 };
 
+export const Closable: StoryFn<typeof PillNext> = () => {
+  const [colors, setColor] = useState(shortColorData);
+
+  const removeColor = (color: string) => {
+    const filteredColors = colors.filter((colorItem) => colorItem !== color);
+    console.log(`Closed ${color}`);
+    setColor(filteredColors);
```

Comment:
```suggestion
    setColor(old => old.filter(colorItem => colorItem !== color));
```
We should use the async safe way of setting state here, just incase this example is copied.